Update EDM inputs to HLT-Validation tests [`10_2_X` only]

missirol opened the pull request:
#### PR description:

This PR is specific to the `CMSSW_10_2_X` release cycle.

Some of the HLT-Validation tests running in IBs started to fail due to the unavailability of EDM files used as input to these tests (see [IB dashboard](https://cmssdt.cern.ch/SDT/html/cmssdt-ib/#/ib/CMSSW_10_2_X), and https://github.com/cms-sw/cmssw/issues/40013).

This PR updates the path to these files, ~~to use copies kept in the TSG area on EOS.~~ which will now rely on the cms-bot cache (see discussion in https://github.com/cms-sw/cmssw/issues/40013 and #40020, where the corresponding updates to `cms-bot` are also mentioned).

In addition, it disables the HLT-integration-tests part of the HLT-Validation tests for scram-archs in SLC6, which are incompatible with the most recent version of ConfDB-v2 (see https://github.com/cms-sw/cmssw/issues/40013#issuecomment-1326250032).

#### PR validation:

None.

#### If this PR is a backport, please specify the original PR and why you need to backport that PR. If this PR will be backported, please specify to which release cycle the backport is meant for:

N/A

cmsbuild commented:
A new Pull Request was created by  @missirol (Marino Missiroli) for CMSSW_10_2_X.

It involves the following packages:

- HLTrigger/Configuration (**hlt**)
- Utilities/ReleaseScripts (**core**)


@Martin-Grunewald, @Dr15Jones, @smuzaffar, @makortel, @cmsbuild, @missirol can you please review it and eventually sign? Thanks.
@Martin-Grunewald, @silviodonato, @wddgit this is something you requested to watch as well.
@perrotta, @dpiparo, @rappoccio you are the release manager for this.

cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

missirol commented:
please test

cmsbuild commented:
+1

**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-470f46/28915/summary.html
**COMMIT**: 688c32d00a63ad375582b8e03621f013997af239
**CMSSW**: CMSSW_10_2_X_2022-11-06-0000/slc6_amd64_gcc700
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/40028/28915/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Comparison Summary



**Summary**:
 - No significant changes to the logs found
 - Reco comparison results: 0 differences found in the comparisons
 - DQMHistoTests: Total files compared: 31
 - DQMHistoTests: Total histograms compared: 3007546
 - DQMHistoTests: Total failures: 2
 - DQMHistoTests: Total nulls: 0
 - DQMHistoTests: Total successes: 3007354
 - DQMHistoTests: Total skipped: 190
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: 0.0 KiB( 30 files compared)
 - Checked 129 log files, 28 edm output root files, 31 DQM output files
 - [TriggerResults](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_10_2_X_2022-11-06-0000+470f46/53996/triggerResults): no differences found

missirol committed:
Update EDM input files of HLT-Validation tests

missirol commented:
please test

See https://github.com/cms-sw/cmssw/pull/40030#issuecomment-1325478219.

cmsbuild commented:
+1

**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-470f46/29215/summary.html
**COMMIT**: b6d2b06a765f021d8f49c80b52322ca95f084e57
**CMSSW**: CMSSW_10_2_X_2022-11-20-2300/slc6_amd64_gcc700
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/40028/29215/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Comparison Summary



**Summary**:
 - No significant changes to the logs found
 - Reco comparison results: 0 differences found in the comparisons
 - DQMHistoTests: Total files compared: 31
 - DQMHistoTests: Total histograms compared: 3007546
 - DQMHistoTests: Total failures: 2
 - DQMHistoTests: Total nulls: 0
 - DQMHistoTests: Total successes: 3007354
 - DQMHistoTests: Total skipped: 190
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: 0.0 KiB( 30 files compared)
 - Checked 129 log files, 28 edm output root files, 31 DQM output files
 - [TriggerResults](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_10_2_X_2022-11-20-2300+470f46/54274/triggerResults): no differences found

missirol committed:
Skip HLT-integration tests in SLC6

HLT-integration tests cannot run with SLC6 architectures,
due to an incompatibility with the latest .jar files of ConfDB-v2.
For further details, see https://github.com/cms-sw/cmssw/issues/40013#issuecomment-1325571973

cmsbuild commented:
Pull request #40028 was updated. @Martin-Grunewald, @Dr15Jones, @smuzaffar, @makortel, @cmsbuild, @missirol can you please check and sign again.

missirol commented:
please test

See https://github.com/cms-sw/cmssw/issues/40013#issuecomment-1326250032.

cmsbuild commented:
+1

**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-470f46/29232/summary.html
**COMMIT**: d5fff4907952a1d48d3341d3541b7bc0ae72f42a
**CMSSW**: CMSSW_10_2_X_2022-11-20-2300/slc6_amd64_gcc700
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/40028/29232/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Comparison Summary



**Summary**:
 - No significant changes to the logs found
 - Reco comparison results: 0 differences found in the comparisons
 - DQMHistoTests: Total files compared: 31
 - DQMHistoTests: Total histograms compared: 3007546
 - DQMHistoTests: Total failures: 5
 - DQMHistoTests: Total nulls: 0
 - DQMHistoTests: Total successes: 3007351
 - DQMHistoTests: Total skipped: 190
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: 0.0 KiB( 30 files compared)
 - Checked 129 log files, 28 edm output root files, 31 DQM output files
 - [TriggerResults](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_10_2_X_2022-11-20-2300+470f46/54292/triggerResults): no differences found

missirol commented:
+hlt

This PR fixes the HLT-Val tests currently failing in `10_2_X` IBs (access EDM files from cms-bot cache + disable HLT-integration tests in SLC6).

The update of 1 input file of the `addOnTests` requires the signature of @cms-sw/core-l2 .

smuzaffar commented:
+core

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_10_2_X IBs (tests are also fine) and once validation in the development release cycle CMSSW_13_0_X is complete. This pull request will now be reviewed by the release team before it's merged. @perrotta, @dpiparo, @rappoccio (and backports should be raised in the release meeting by the corresponding L2)

perrotta commented:
+1
